src/main/java/domain/gamer/Gamer.java

```diff
@@ -0,0 +1,38 @@
+package domain.gamer;
+
+import domain.card.Card;
+import domain.card.PlayingCards;
+
+public class Gamer {
+    final PlayingCards playingCards;
+    final String name;
+
+    public Gamer(PlayingCards playingCards, String name) {
+        this.playingCards = playingCards;
+        this.name = name;
+    }
+
+    public void addCard(Card card) {
```

Comment:
외부에서 Deck의 카드를 뽑아서 전달 하는 것이 아니라
Gamer에게 Deck을 전달해 직접 카드를 뽑게 하면 어떨까요? 

```suggestion
    public void addCard(Deck deck) {
        playingCards.add(deck.handOutCard()); 
    }
```